yal/main.py

```diff
@@ -147,6 +147,20 @@ async def process_message(self, message):
                 message.transport_metadata, "message", "button", "payload"
             )
 
+            whatsapp_delivery_failed = self.user.metadata.get(
+                "whatsapp_delivery_failed", "False"
+            )
+
+            if whatsapp_delivery_failed == "True":
```

Comment:
Making this case sensitive might cause problems later on if someone updates the flow